components/Heading/Heading.js

```diff
@@ -1,12 +1,43 @@
 import styled from "styled-components";
-import ArrowBackIcon from "../../public/icons/arraw_left";
+import ArrowBackIcon from "../../public/icons/Chevron_left";
 import LoginIcon from "../../public/icons/login";
+import { useRouter } from "next/router";
+
+export default function Heading({ children, previousLevelUrl, isShowButton }) {
+  const router = useRouter();
+
+  if (!router.isReady) {
+    return <h1>loading</h1>;
+  }
+
+  function handleGoBackPage() {
+    router.push(previousLevelUrl);
+  }
+
+  return (
+    <StyledHeader>
+      <StyledButton
+        type="button"
+        onClick={handleGoBackPage}
+        aria-label="go back to the last page"
+        isShowButton={isShowButton}
+      >
+        <ArrowBackIcon />
+      </StyledButton>
+      <h1>{children}</h1>
```

Comment:
wouldnt recommend that, because this can run into hydration problems.
So like <h1> <p> </p> </h1> can cause this error